index.html

```diff
@@ -8,51 +8,232 @@
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="link">
+              <a href="">See Project</a>
+            </div>
+          </div>
+        </div>
+      </section>
+      <!-- end of tonic -->
+            <section class="works">
+        <div class="cards">
+          <img src="multi.svg" alt="" />
+          <div class="projects">
+            <div class="project-section">
+              <h1 class="project-title">Multi post stories</h1>
+              <div class="extra-info">
+                <p class="proj-name">CANOPY</p>
+                <img src="Circle.svg" alt="" />
+                <p class="job-title">Back End dev</p>
+                <img src="Circle.svg" alt="" />
+                <p class="job-title">2015</p>
+              </div>
+            </div>
+            <p class="details">
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required.
+            </p>
+            <ul class="lang-tags">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <div class="link">
+              <a href="">See Project</a>
+            </div>
+          </div>
+        </div>
+      </section>
+      
+      <!-- this is the about page -->
+
+      <section class="about">
+        <h1 class="title">About Myself</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hestiate to contact
+          me.
+        </p>
+
+        <div class="social-media">
+          <h5>LETS CONNECT</h5>
+          <ul>
+            <li>
+              <a href=""><img src="twitter.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="linked.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="meduimsvg.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="github.svg" alt="" /></a>
+            </li>
+            <li>
+              <a href=""><img src="angellist.svg" alt="" /></a>
+            </li>
+          </ul>
+        </div>
+        <div class="link">
+          <a href="">Get my Resume</a>
+        </div>
+       
+          <ul class="btn">
+            <li class="languages">  Language  <img src="Disabled.svg" alt="" /></li>  
+          </ul>
+          
+              <ul class="lang-list">
+                <li class="js"> <img src="js.png" alt=""> JavaScript</li>
+                <li class="js"> <img src="ht.svg" alt=""> HTML</li>
+                <li class="js"> <img src="Ellipse.svg" alt="">Css</li>
+              </ul>
+              
+            <h6 class="frame">Framework <img src="Icon.svg" alt="button"> </h6>
+            <div class="sep"></div>
```

Comment:
- [ ] Kindly make sure you have space after all the language icons so that it looks clear. Basically, the icons and the text should have space in between as per the design shown in the attached loom video.